FreshRSS/FreshRSS
# Improved: Centralized CSS for notifications

It does not make sense to have all CSS rules for the notifications in the dedicated themes.

Changes proposed in this pull request:

- (S)CSS


How to test the feature manually:

1. save some configurations
2. see the notifications


Pull request checklist:
